install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,67 +11,68 @@
 toc::[]
 
 == Overview
-You can provision your OpenShift cluster with
```

Comment:
Do we want to use words like "You" in this doc? I removed uses of "you", "your",  etc. thinking it was a bit more formal...